src/main/flight/pid.c

```diff
@@ -638,6 +637,17 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
 
             pidData[axis].D = pidCoefficient[axis].Kd * delta * tpaFactor;
 
+
+            const float pid_ff =
```

Comment:
Let's call it `pidFeedForward(ed)`, also obeying `camelCase` for local variables (at least in this file).